src/sentry/api/helpers/teams.py

```diff
@@ -1,8 +1,23 @@
+from typing import Optional
+
 from rest_framework.exceptions import PermissionDenied
 
 from sentry.api.utils import InvalidParams
 from sentry.auth.superuser import is_active_superuser
 from sentry.models import Team, TeamStatus
+from sentry.models.organizationmember import OrganizationMember
+from sentry.models.organizationmemberteam import OrganizationMemberTeam
+
+
+def is_team_admin(org_member: OrganizationMember, team: Optional[Team] = None) -> bool:
```

Comment:
This _may_ make sense to make it a helper function for the model `OrganizationMemberTeam`

Reply:
that's prob a good suggestion, I'll move it there

Reply:
Actually I think I have to leave it there, o/w it runs into a circular import error between `Organization` and `OrganizationMemberTeam` model files